src/DynamoApplications/StartupUtils.cs

```diff
@@ -150,124 +146,9 @@ public static CommandLineArguments Parse(string[] args)
-                "this option is only supported when run from DynamoSandbox", c => commandFilePath = c)
-                .Add("l=|L=", "Running Dynamo under a different locale setting", l => locale = l)
-                .Add("v=|V=", "Verbose, Instruct Dynamo to output all evalautions it performs to an xml file at this path", v => verbose = v)
-                .Add("x|X", "When used in combination with the 'O' flag, opens a .dyn file from the specified path and converts it to .json." +
-                "File will have the .json extension and be located in the same directory as the original file.", x => convertFile = x != null)
-                .Add("h|H|help", "Get some help", h => showHelp = h != null)
-                .Add("g=|G=|geometry", "Geometry, Instruct Dynamo to output geometry from all evaluations to a json file at this path", g => geometryFilePath = g)
-                .Add("i=|I=|import", "Import, Instruct Dynamo to import an assembly as a node library. This argument should be a filepath to a single .dll" +
-                " - if you wish to import multiple dlls - use this flag multiple times: -i 'assembly1.dll' -i 'assembly2.dll' ", i => importPaths.Add(i))
-                .Add("gp=|GP=|geometrypath=|GeometryPath=", "relative or absolute path to a directory containing ASM. When supplied, instead of searching the hard disk for ASM, it will be loaded directly from this path.", gp => asmPath = gp)
-                .Add("k|K|keepalive", "Keepalive mode, leave the Dynamo process running until a loaded extension shuts it down.", k => keepAlive = k != null)
-                .Add("nc|NC|noconsole", "Don't rely on the console window to interact with CLI in Keepalive mode", nc => noConsole = nc != null)
-                .Add("ud=|UD=|userdata", "Specify user data folder to be used by PathResolver with CLI", ud => userDataFolder = ud)
-                .Add("cd=|CD=|commondata", "Specify common data folder to be used by PathResolver with CLI", cd => commonDataFolder = cd)
-                .Add("hn=|HN=|hostname", "Identify Dynamo variation associated with host", hn => hostname = hn)
-                .Add("si=|SI=|sessionId", "Identify Dynamo host analytics session id", si => sessionId = si)
-                .Add("pi=|PI=|parentId", "Identify Dynamo host analytics parent id", pi => parentId = pi)
-                .Add("da|DA|disableAnalytics|DisableAnalytics", "Disables analytics in Dynamo for the process liftime", da => disableAnalytics = da != null)
-                .Add("cr=|CR=|cerLocation", "Specify the crash error report tool location on disk ", cr => cerLocation = cr)
-                .Add("s|S|service mode", "Service mode, bypasses certain Dynamo launch steps for maximum startup performance", s => serviceMode = s != null);
-                optionsSet.Parse(args);
-
-                if (showHelp)
-                {
-                    ShowHelp(optionsSet);
```

Comment:
is the `ShowHelp` bool ever used? It looks like the method was removed?

Reply:
@mjkkirschner No, it's not being used anymore, the method just was used for .NET 4.8.

Reply:
so I am confused, what does the `-h` flag do now, and why set a bool?

Reply:
When I started this task I had to compare the flags for .NET 4.8 against .NET 6, I found that the ShowHelp flag was missing in .NET 6, so, I had to add it, and the -h flag I think is doing the same as in .NET 4.8, I defined it as a bool because in .NET 4.8 also was a bool.

Reply:
@JimmySanchezGlobant
please don't add any flags that have no use.

After your PR will the CLI have the ability to show help or not? it appears to be a built in functionality of the command line parser library using `--help` ?

Reply:
@mjkkirschner To be honest with you I don't know if the CLI will show help or not with my PR, but if you consider it I could remove this new flag.

Reply:
Can you try testing it locally? You can use a net4.8 build to avoid net6 complications. Or you can try adding a test to the test project you have been modifying.

Reply:
@JimmySanchezGlobant did you find an answer to Mike's question here?

Reply:
Not yet, I'm still testing.

Reply:
According to what @twastvedt says the ShowHelp flag is completely unnecessary, so, the solution is to remove it.

Reply:
Solved.